### PR TITLE
Support parquet file format.

### PR DESCRIPTION
**Pull request by stephenplusplus**
Is this it? :)

### CODE REVIEW

**Comment by stephenplusplus** [comment]
cc @tswast

**Comment by stephenplusplus** [comment]
Thanks @tswast!

@callmehiphop would you mind running the sample tests to see if they pass?

File: samples/tables.js [hunk 1]
```diff
@@ -240,6 +240,74 @@ function loadLocalFile(datasetId, tableId, filename, projectId) {
   // [END bigquery_load_from_file]
 }
 
+function loadParquetFromGCS(datasetId, tableId, projectId) {
```
Comment (tswast) [inline samples/tables.js]: Thanks!

**Comment by callmehiphop** [comment]
@stephenplusplus sorry, I somehow missed your request. I just ran the samples test and they do not pass for me on my local machine.

```
✖ tables › should load a GCS Parquet file with explicit schema 

  1 test failed
  2 tests skipped

  tables › should load a GCS Parquet file with explicit schema

  /Users/taco/projects/nodejs-bigquery/samples/system-test/tables.test.js:21

   20: const storage = require(`@google-cloud/storage`)();      
   21: const test = require(`ava`);                             
   22: const tools = require(`@google-cloud/nodejs-repo-tools`);

  Value must match expression:

  ''

  Regular expression:

  /completed\./

  output
  => ''
```

**Pull request merged**